types/angular-animate/index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: http://angularjs.org
 // Definitions by: Michel Salib <https://github.com/michelsalib>, Adi Dahiya <https://github.com/adidahiya>, Raphael Schweizer <https://github.com/rasch>, Cody Schaaf <https://github.com/codyschaaf>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 2.3
```

Comment:
`@types/angular` technically doesn't require TypeScript 2.3, only TypeScript 2.1. It has a TypeScript 2.3 header so that merging tests with `@types/jquery` v3 will work. I thought I documented it somewhere but apparently not. Could you add that as a note in `types/angular/index.d.ts`?